rbi/rootfs-img/image-check/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM registry.fedoraproject.org/fedora:34-x86_64
+
+LABEL maintainer="Ding Ma <ding.ma@linux.alibaba.com>"
+
+RUN [ -n "$http_proxy" ] && sed -i '$ a proxy='$http_proxy /etc/dnf/dnf.conf ; true
+
+RUN dnf install -y qemu-img-2:5.2.0-8.fc34.x86_64 \
```

Comment:
ditto

Reply:
Those version info has been deleted. And version format in `rootfs-img/rootfs/patch/rootfs-builder/centos/config.sh` has been changed too.

Reply:
https://github.com/alibaba/inclavare-containers/pull/1255/commits/838ef63bc149424ba4f60150d075ab633f523673